cmd/snap-mgmt/snap-mgmt.sh.in

```diff
@@ -119,6 +125,11 @@ purge() {
         rm -f "/etc/systemd/system/multi-user.target.wants/$unit"
     done
 
+    # snapd session-agent
+    rm -f /etc/systemd/user/snapd.session-agent.socket
```

Comment:
Wouldn't this be done by the package manager?

Reply:
Hmm, I actually wonder about this now.

Why are we installing to etc? That should be done only to /usr/lib, and snapd should probably only write to /usr/local, in the case of re-execution. We should not be shipping a file that goes here.

Reply:
I've looked at the RPM specs and PKGBUILD, none of those install session-agent bits into /etc. It's unclear how those files ended up at that location.

Reply:
We write our files there on re-exec, this is why the script removes them. I wonder if our packaging should change so that package-managed files go to lib instead.